chevah_build

```diff
@@ -159,6 +159,13 @@ case $OS in
         export CC="cc"
         export CXX="CC"
         export MAKE="gmake"
+
+        # The package on Solaris 10 does not compile with CC, we use GCC.
```

Comment:
why they don't compile?
 they used to work with CC and Sun Studio... this is what we had for the previous solaris 10 and solaris 11 on both x86 and sparc